src/presentation/cli/controller/runtime.go

```diff
@@ -0,0 +1,256 @@
+package cliController
+
+import (
+	"errors"
+	"log/slog"
+	"strconv"
+	"strings"
+
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	infraHelper "github.com/speedianet/os/src/infra/helper"
+	internalDbInfra "github.com/speedianet/os/src/infra/internalDatabase"
+	cliHelper "github.com/speedianet/os/src/presentation/cli/helper"
+	"github.com/speedianet/os/src/presentation/service"
+	sharedHelper "github.com/speedianet/os/src/presentation/shared/helper"
+	"github.com/spf13/cobra"
+)
+
+type RuntimeController struct {
+	runtimeService *service.RuntimeService
+}
+
+func NewRuntimeController(
+	persistentDbService *internalDbInfra.PersistentDatabaseService,
+) *RuntimeController {
+	return &RuntimeController{
+		runtimeService: service.NewRuntimeService(persistentDbService),
+	}
+}
+
+func getHostname(hostnameStr string) (hostname valueObject.Fqdn, err error) {
+	primaryVhost, err := infraHelper.GetPrimaryVirtualHost()
+	if err != nil {
+		return hostname, errors.New("PrimaryVirtualHostNotFound")
+	}
+
+	hostname = primaryVhost
+	if hostnameStr != "" {
+		return valueObject.NewFqdn(hostnameStr)
+	}
+
+	return hostname, nil
+}
+
+func (controller *RuntimeController) ReadPhpConfigs() *cobra.Command {
+	var hostnameStr string
+
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "GetPhpConfigs",
+		Run: func(cmd *cobra.Command, args []string) {
+			hostname, err := getHostname(hostnameStr)
+			if err != nil {
+				cliHelper.ResponseWrapper(false, err.Error())
+			}
+
+			requestBody := map[string]interface{}{
+				"hostname": hostname.String(),
+			}
+
+			cliHelper.ServiceResponseWrapper(
+				controller.runtimeService.ReadPhpConfigs(requestBody),
+			)
+		},
+	}
+
+	cmd.Flags().StringVarP(&hostnameStr, "hostname", "n", "", "Hostname")
+	return cmd
+}
+
+func parsePhpModules(rawPhpModules []string) []entity.PhpModule {
+	modules := []entity.PhpModule{}
+	for _, rawModule := range rawPhpModules {
+		rawModuleParts := strings.Split(rawModule, ":")
+		rawModulePartsLength := len(rawModuleParts)
+		if rawModulePartsLength == 0 {
+			slog.Debug("PhpModuleEmpty", slog.String("module", rawModule))
+			continue
+		}
+
+		moduleName, err := valueObject.NewPhpModuleName(rawModuleParts[0])
+		if err != nil {
+			slog.Debug("InvalidPhpModuleName", slog.Any("name", rawModuleParts[0]))
+			continue
+		}
+
+		moduleStatus := true
+		if rawModulePartsLength > 1 {
+			var err error
+			moduleStatus, err = sharedHelper.ParseBoolParam(rawModuleParts[1])
```

Comment:
Lol, here you used the parse bool param. Why did you cast to bool on the api controller then?

Reply:
Changed to use `voHelper.InterfaceToBool()` instead type assertation in **API** controller.